index.html

```diff
@@ -46,7 +46,7 @@
   </header>
   <main class="main-index">
     <h1 class="visually hidden">Cайт города Седона</h1>
-    <img src="#" alt="Изображение гор и надпись: welcome to the gorgeous Sedona because the Grand Canyon sucks!">
+    <img src="images/backgrounds-and-advantages/index-background.jpg" width="1200" height="485" alt="Изображение гор и надпись: welcome to the gorgeous Sedona because the Grand Canyon sucks!">
```

Comment:
фоновое изображение не может быть контентным, оно декоративное. Здесь необходимо подключить изображение - welcome to the gorgeous Sedona because the Grand Canyon sucks!